modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/remote_services.js

```diff
@@ -360,30 +360,71 @@ AUI.add(
 				_invokeResourceURL: function(params) {
 					var instance = this;
 
```

Comment:
Hey @dustinryerson,

I've just discussed with Adam about your pull and he would like to know if you've considered to fix this issue using a POST request, once a POST would avoid the URI size limitation.